vyper/exceptions.py

```diff
@@ -47,6 +49,18 @@ def __str__(self):
         return self.message
 
 
+class PythonSyntaxException(ParserException):
+    """
+    Conversion from error using ast.parse()
+    """
+    def __init__(self, syntax_error: SyntaxError, source_code: str):
+        item = types.SimpleNamespace()  # TODO: Create an actual object for this
```

Comment:
TODO is here in case this object is more broadly useful (which I don't think it is)

Would also just be better if it were a static object